.github/workflows/dependencies.yml

```diff
@@ -28,23 +28,23 @@ jobs:
     - name: Run Checks
       run: |
         python tests/scripts/check_pip.py missing cvxpy
-        python tests/scripts/check_pip.py contains scikit-learn
+        python tests/scripts/check_pip.py installed scikit-learn
         python tests/scripts/import_all.py
     - name: Install cvxpy
       run: |
         python -m pip install -e ".[cvxpy]"
     - name: Run Checks
       run: |
-        python tests/scripts/check_pip.py contains cvxpy scikit-learn
+        python tests/scripts/check_pip.py installed cvxpy scikit-learn
         python tests/scripts/import_all.py
     - name: Install All
       run: |
         python -m pip install -e ".[all]"
     - name: Run Checks
       run: |
-        python tests/scripts/check_pip.py contains cvxpy scikit-learn
+        python tests/scripts/check_pip.py installed cvxpy formulaic patsy scikit-learn umap-learn
     - name: Docs can Build
       run: |
         sudo apt-get update && sudo apt-get install pandoc
         python -m pip install -e ".[docs]"
-        sphinx-build doc docs
+        mkdocs build
```

Comment:
> as a final step, could we add a CI file like this one?

@koaning how's that different from the one we have here?

Reply:
Ah! I somehow missed that. LGTM then :)